src/main/java/com/minecolonies/coremod/entity/ai/citizen/shepherd/EntityAIWorkShepherd.java

```diff
@@ -0,0 +1,504 @@
+                sheep.getPosition().getY(), sheep.getPosition().getZ());
+              return range <= MAX_DISTANCE_FROM_HUT;
+          }
+        ));
+
+        return SHEPHERD_DECIDE;
+    }
+
+    /**
+     * Creates a simple area around the Shepherd Hut used for AABB calculations for finding sheep
+     *
+     * @return The AABB
+     */
+    private AxisAlignedBB getTargetableArea()
+    {
+        final Structures.StructureName sn =
+          new Structures.StructureName(Structures.SCHEMATICS_PREFIX, getOwnBuilding().getStyle(), getOwnBuilding().getSchematicName() + getOwnBuilding().getBuildingLevel());
+
+        final String structureName = sn.toString();
+
+        final StructureWrapper wrapper = new StructureWrapper(world, structureName);
+        wrapper.rotate(getOwnBuilding().getRotation(), world, getOwnBuilding().getLocation(), getOwnBuilding().isMirrored() ? Mirror.FRONT_BACK : Mirror.NONE);
+
+        final BlockPos pos = getOwnBuilding().getLocation();
+        wrapper.setPosition(pos);
+
+        final int x1 = wrapper.getPosition().getX() - wrapper.getOffset().getX() - 1;
+        final int z1 = wrapper.getPosition().getZ() - wrapper.getOffset().getZ() - 1;
+        final int x3 = wrapper.getPosition().getX() + (wrapper.getWidth() - wrapper.getOffset().getX());
+        final int z3 = wrapper.getPosition().getZ() + (wrapper.getLength() - wrapper.getOffset().getZ());
+        final int y1 = getOwnBuilding().getLocation().getY() - 10;
+        final int y3 = getOwnBuilding().getLocation().getY() + 10;
+
+        return new AxisAlignedBB(x1, y1, z1, x3, y3, z3);
+    }
+
+    /**
+     * Shears a sheep, with a chance of dying it!
+     *
+     * @return the wanted AiState
+     */
+    private AIState shearSheep()
+    {
+        job.removeDeadSheep();
+
+        final List<EntitySheep> sheepies = job.getSheep();
+
+        if (sheepies.isEmpty())
+        {
+            return SHEPHERD_SEARCH_FOR_SHEEP;
+        }
+
+        final EntitySheep sheep = sheepies.stream().filter(sheepie -> !sheepie.getSheared()).findFirst().orElse(null);
+
+        if (!equipShears())
+        {
+            return PREPARING;
+        }
+
+        if (worker.getHeldItemMainhand() != null && sheep != null)
+        {
+            final List<ItemStack> items = sheep.onSheared(worker.getHeldItemMainhand(),
+              worker.worldObj,
+              worker.getPosition(),
+              net.minecraft.enchantment.EnchantmentHelper.getEnchantmentLevel(net.minecraft.init.Enchantments.FORTUNE, worker.getHeldItemMainhand()));
+
+            dyeSheepChance(sheep);
+
+            worker.getHeldItemMainhand().damageItem(1, worker);
+
+            worker.addExperience(EXP_PER_SHEEP);
+
+            for (final ItemStack item : items)
+            {
+                worker.getInventoryCitizen().addItemStackToInventory(item);
+            }
+        }
+
+        //setDelay(40);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3ACommentedOutCodeLine)